css/style.css

```diff
@@ -1,96 +1,357 @@
+    font-weight: 400;
+    font-size: 56px;
+    line-height: 64px;
+    letter-spacing: 0.3px;
+    color: #ff6b00;
+  }
 
-p {
-  width: 327px;
-  height: 123px;
-  font-family: Arial, Helvetica, sans-serif;
-  font-style: normal;
-  font-weight: 400;
-  font-size: 16px;
-  line-height: 24px;
-  color: #fff;
-  flex: none;
-  flex-grow: 0;
-}
+  h2 {
+    width: 327px;
+    height: 27px;
+    font-family: 'Arial, Helvetica', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 24px;
+    line-height: 32px;
+    display: flex;
+    align-items: center;
+    color: #fff;
+  }
 
-.menu {
-  display: flex;
-  flex-flow: nowrap;
-  list-style: none;
-  justify-content: space-between;
-}
+  p {
+    width: 327px;
+    height: 123px;
+    font-family: ' Arial, Helvetica', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    color: #fff;
+    flex: none;
+    flex-grow: 0;
+  }
+
+  .menu {
+    display: flex;
+    flex-flow: nowrap;
+    list-style: none;
+    justify-content: space-between;
+  }
+
+  .social {
+    list-style: none;
+    display: flex;
+    justify-content: flex-start;
+    gap: 10px;
+    vertical-align: bottom;
+    line-height: 1.22;
+    width: 200px;
+  }
+
+  hr {
+    width: 327px;
+    height: 0;
+    border: 1px solid #212121;
+    flex: none;
+    flex-grow: 0;
+  }
+
+  .recent-works {
+    width: 100%;
+    margin: 0 24px;
+  }
+
+  .recent-works h1 {
+    font-family: 'Crete Round', sans-serif;
+    font-style: normal;
+    font-size: 40px;
+    font-weight: 400;
+    line-height: 52px;
+    letter-spacing: 0;
+    text-align: left;
+    height: 83px;
+    width: 100%;
+    align-self: flex-start;
+  }
+
+  .work-list {
+    display: grid;
+    grid-template-rows: auto auto auto auto auto auto;
+    height: auto;
+    gap: 24px;
+  }
+
+  .work-item {
+    margin-top: 80px;
+  }
+
+  .work-item img {
+    display: block;
+
+    /* width: 100%; */
+    margin-bottom: 24px;
+  }
+
+  .work-item p {
+    color: black;
+    height: 147px;
+
+    /* width: 100%; */
+    font-family: 'Inter', sans-serif;
+    font-size: 16px;
+    font-weight: 400;
+    line-height: 24px;
+    letter-spacing: 0;
+    text-align: left;
+  }
+
+  .work-item ul {
+    display: flex;
+    justify-content: flex-start;
+    gap: 12px;
+    padding-left: 0;
+    margin-bottom: 24px;
+  }
+
+  .work-item li {
+    border: 1px solid #8993a4;
+    padding: 10px;
+    color: #091e42;
+    list-style: none;
+  }
+
+  .work-item a {
+    background: #ff6b00;
+    padding: 12px;
+    width: 124px;
+    height: 48px;
+    align-items: center;
+    text-decoration: none;
+    color: white;
+    font-weight: 700;
+    font-size: 17px;
+    line-height: 24px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    letter-spacing: 0.03em;
+  }
+
+  .work-item-alt {
+    width: 100%;
+    background-image: url('/assets/images/bg.png');
```

Comment:
- [ ] [Required] Please make sure all the images are rendered properly. This URL doesn't render Images. It's always a good practice to check your links and images before submission. 
You can use `background-image: url('../assets/images/bg.png');` to render image. 
![image_Screenshot from 2022-05-29 01-28-15](https://user-images.githubusercontent.com/24191094/170844754-946c56a3-cb56-46cf-86c6-d956e9ad9fd0.png)